### PR TITLE
[ADPRODUCTS-370] Documentation updates with CS data revamp

### PR DESCRIPTION
**Pull request by skahmann3**
Creating Sponsored Tiles dataset documentation in DTMO. Also adding the new datatables to the docs.

### CODE REVIEW

**Comment by skahmann3** [comment]
@rebecca-burwei - created documentation for Tiles, noting the new datasetes. Please review when you get a chance! Though I'll hold off on merging until we know where the new tables are landing in Looker

File: src/datasets/other/sponsored tiles/sponsored_tiles.md
Comment (rebecca-burwei) [inline src/datasets/other/sponsored tiles/sponsored_tiles.md:5]:
```suggestion
Firefox Sponsored Tiles is an advertising-based feature available on the new tab of both desktop and mobile Firefox browsers. This feature populates the new tab's Top Sites section with up to two Sponsored Tiles. These Sponsored Tiles display a company logo and link to the company website. Mozilla's Contile serves the advertisements and may refresh the advertisers every 15 minutes. Each click on a Sponsored Tile generates revenue for Mozilla.
```

File: src/datasets/other/sponsored tiles/sponsored_tiles.md
Comment (rebecca-burwei) [inline src/datasets/other/sponsored tiles/sponsored_tiles.md:19]:
```suggestion
| All Mozillians               | `telemetry.sponsored_tiles_clients_daily`     | TBD                                    | Workhorse dataset for Sponsored Tiles, includes desktop and mobile data. All new Sponsored Tiles metrics are added to this table. Does not include advertiser data. |
```

File: src/datasets/other/sponsored tiles/sponsored_tiles.md
Comment (rebecca-burwei) [inline src/datasets/other/sponsored tiles/sponsored_tiles.md:24]: I think this dataset limited to people in both Revenue and Contextual Services.

**Review by rebecca-burwei — COMMENTED** [review]
Some minor changes, but overall looks good.

**Review by rebecca-burwei — APPROVED** [review]

**Review by badboy — CHANGES_REQUESTED** [review]
Please use an underscore `_` instead of spaces in directory and file names.
Additionally this file is not referenced anywhere, and thus won't be accessible in the book.
You can add it to the table of contents in the `SUMMARY.md` file.

I think addressing these 2 things should then make the tests pass as well.

**Comment by skahmann3** [comment]
Thanks for catching those edits, @badboy. Will you re-review? But please don't merge changes yet, we're waiting for another BQ table to land before finalizing these updates

**Review by badboy — APPROVED** [review]
No blocker from my side then. Merge when you are ready.

**Review by rebecca-burwei — APPROVED** [review]
Looks good!

**Pull request merged**
